packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -124,6 +124,58 @@ const ShowNotifications = ({ is_notifications_visible, notifications_count, togg
     );
 };
 
+const DefaultMobileLinks = ({
+    is_dark_mode,
+    toggleIsTourOpen,
+    is_mf,
+    is_eu,
+    is_eu_country,
+    setIsOnboardingVisited,
+    is_notifications_visible,
+    notifications_count,
+    toggleNotifications,
+    is_in_progress,
+    handleClickCashier,
```

Comment:
Can we wrap this in an observer and get all this from the store here itself rather than passing props